### PR TITLE
Update integration to work with Cyber Triage 2.6

### PR DESCRIPTION
**Pull request by crayy8**
Allow user to specify a Cyber Triage incident and choose the type of data to collect. 

## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Updating the Cyber Triage integration app to allows users to specify the type of data they want to collect. Also allow users to specify and incident name when creating a Cyber Triage session. Cyber Triage 2.6 or later is required to use this integration. 

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

File: Integrations/integration-CyberTriage.yml [hunk 1]
```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
         endpoint = CMD_ARGS['endpoint']
+        scan_options = CMD_ARGS['scan_options']
+        incident_name = CMD_ARGS.get('incident_name','Default') # If no incident name is provided it is put into the "Default" incident
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Please add `incident_name` to the command arguments in integration settings.

File: Integrations/integration-CyberTriage.yml [hunk 2]
```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
         endpoint = CMD_ARGS['endpoint']
+        scan_options = CMD_ARGS['scan_options']
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Please add `scan_options` to the command arguments in integration settings, and please use `.get()` so you won't get Key Error exception in case the user did not pass it.

File: Integrations/integration-CyberTriage.yml [hunk 3]
```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Why did you remove this?

File: Integrations/integration-CyberTriage.yml [hunk 4]
```diff
@@ -180,25 +182,31 @@ script:
...
+        api_data.update({'scanOptions': scan_options})
         api_data.update({'malwareScanRequested': is_hash_upload_on})
         api_data.update({'sendContent': is_file_upload_on})
-        api_data.update({'fastScan': is_fast_scan_on})
         api_data.update({'sendIpAddress': False})
-
         response = make_rest_call(REST_APIS['start_collection'], 'post', api_data)
 
+        if is_ip_valid(endpoint):
+            endpoint_context = { 'IPAddress' : endpoint }
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Please lint properly.

File: Integrations/integration-CyberTriage.yml [hunk 5]
```diff
@@ -180,25 +182,31 @@ script:
...
+        else:
+            endpoint_context = { 'Hostname' : endpoint }
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Please lint properly.

File: Integrations/integration-CyberTriage.yml [hunk 6]
```diff
@@ -64,6 +64,7 @@ script:
     CT_PARAMS = demisto.params()
     CMD_ARGS = demisto.args()
 
+    SCAN_OPTIONS = ['pr','nw','nc','st','sc','ru','co','lo','ns','wb','fs']
```
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: Please add whitespace after each comma.

File: Integrations/integration-CyberTriage.yml (hunk 3 above)
Comment (crayy8) [reply Integrations/integration-CyberTriage.yml]: Fast scan is no longer supported by Cyber Triage. It was replaced with scanOptions.

File: Integrations/integration-CyberTriage.yml (hunk 2 above)
Comment (crayy8) [reply Integrations/integration-CyberTriage.yml]: scan_options has a default value in Demisto so .get() was not needed.

Comment (Itay4) [reply Integrations/integration-CyberTriage.yml]: I see, well `scan_options` is not defined in integration settings, so couldn't see it has default.

File: Integrations/integration-CyberTriage.yml (hunk 3 above)
Comment (Itay4) [reply Integrations/integration-CyberTriage.yml]: I understand, but removing it will break backward compatibility.
Does `fs` in `scanOptions` replace it?

File: Integrations/integration-CyberTriage.yml (hunk 1 above)
Comment (Itay4) [inline Integrations/integration-CyberTriage.yml]: i think its better to give the argument the default value `Default` instead in the code, the use will see that this is the default
please add whitespace after the comma
